src/components/nodes/pages/summer-course/summer-course-page.tsx

```diff
@@ -142,7 +143,13 @@ const SummerCoursePage = async ({node, ...props}: Props) => {
       <div className="centered">
         <RelatedCourses objectId={node.id} />
       </div>
-      <div>{/* Apply Now Link */}</div>
+      <div className="centered flex flex-col items-end">
+        <div className="w-full md:w-1/2">
+          <ApplyNowLink href="/courses">
```

Comment:
I don't believe that's the correct location. The masthead on the current site goes to `/apply-now`

Reply:
Ah, I had put `/courses` as a placeholder for now. I'll follow up with Sara in the email thread to confirm the URL destination 😄  Thanks Mike!